src/vaadin-tabs.ts

```diff
@@ -0,0 +1,31 @@
+import { customElement } from 'lit-element';
+import { TabsBase } from './vaadin-tabs-base';
+import { TabsMixin } from './vaadin-tabs-mixin';
+import './vaadin-tab';
+
+/**
+ * `<vaadin-tabs>` is a Web Component for easy switching between different views.
+ *
+ * @attr {start | none | both} overflow - Attribute set depending on whether the items fit into viewport.
```

Comment:
Should we mention `@attr {horizontal | vertical} orientation` as well?

Reply:
Currently the `orientation` property is picked up from `OrientationMixin`.

The `@attr` is currently used for state attributes (that element sets on itself).
So I don't think we need to list `orientation` here for now.